### DISCUSSION TITLE
Yaesu FT-710 rig support in QLog

### DISCUSSION THREAD

**Discussion by faocrates** [foldynl/QLog#344] [Q&A]
Using QLog 0.34.0 the Yaesu FT-710 is not available in the drop down with the list of Equipment > Rigs. Is this something that may get supported soon? Is there by any chance \ workaround config I can use for now until it becomes available?

**Comment by foldynl** [comment]
QLog supports all rigs that are provided by following external back-ends: Hamlib, Omnirig (v1/v2), TCI.

If you are talking about Yaesu FT-710 support in Hamlib, then the rig was introduced in Hamlib version 4.5 ([more Changelog](https://github.com/Hamlib/Hamlib/blob/4.5/NEWS)).

You did not write which distribution you use, but QLog on Windows/MacOs already uses v4.5.5. Flatpak for Linux also includes hamlib 4.5.5. Other packages depend on your distribution. As far as I know, Hamlib 4.5 is included, for example, in ubuntu from version 23.04.

Unfortunately, I don't know any workaround for it now. Maybe google answers your question better.

**Reply by faocrates** [reply]
Thank you for your response, I'm using Linux Mint here. I now used the QLog Flatpak for my install and as suggested the FT-710 is now an available option. I've connected FT-710 to the laptop with a USB cable, I have not found any guidance on what settings are needed, so tried some options (see screenshot) which give the error: Rig Error: Get Frequency Error, Error Detail: rig_get_freq(2130): vfo_opt=0, model=1049

Help is needed at this point, if there are any suggestions these are much appreciated.

![Screenshot from 2024-04-06 15-32-22](https://github.com/foldynl/QLog/assets/75755967/c7b064fa-6b11-4ffa-8c09-6993bb124ea5)
![Screenshot from 2024-04-06 15-35-35](https://github.com/foldynl/QLog/assets/75755967/3b7733f4-61c5-4bca-910b-7325c87e1bec)

**Reply by foldynl** [reply]
from google:

based on [this](https://www.youtube.com/watch?v=ZH_zG5fh7-Q), you should use Data Bits 8, Stop Bits 2, Baud Rate 38400. Also check the port whether ttyACM0 is CAT port.

Based on [this](https://support.hamradiodeluxe.com/support/solutions/articles/51000337439-support-for-yaesu-ft-710), you can also use FT-991A as a RigModel - without any guarantee of success.

Unfortunately, I don't have the FT-710 therefore I cannot test it.

**Comment by TecnoTalarn** [comment]
![image](https://github.com/foldynl/QLog/assets/61935185/e9e47b11-b13a-44da-9a4d-f4047092284a)
Hello, I use the Yaesu FTDX10 in order to run my FT710.
The FT710 option not work well for my.

**Comment by faocrates** [comment]
Thank you for your responses which have helped me to resolve this, I now have a successful connection to the FT-710. My final working settings can be seen in the screenshot below. (I struggled for a bit to find which is the correct port to use, installing flrig helped me with that)

![image](https://github.com/foldynl/QLog/assets/75755967/b66a7c43-9843-44e0-8dfe-165ba2ced86d)

**Comment by TecnoTalarn** [comment]
Hello again
I use a link to usb port by-id to connect to radio. This port not change if another usb to serail is connected to radio first to the yaesu.

![image](https://github.com/foldynl/QLog/assets/61935185/3d08321d-afe5-4e1d-b51f-022a2f690149)
